base/templates/base/minutes_and_reports_inner_table.html

```diff
@@ -1,10 +1,10 @@
-              {% for date, items in table.items %}
-                <tr>
-                  <td>{{date}}</td>
-                  <td>
+              <dl>
+                {% for date, items in table.items %}
+                  <dt>{{date}}</dt>
+                  <dd>
                   {% for item in items %}
-                    <a href="{{item.url}}">{{item.summary}}</a><br/>
+                    <a href="{{item.url}}">{{item.summary}}</a>
                   {% endfor %}
-                  </td>
-                </tr>
-              {% endfor %}
+                  </dd>
```

Comment:
This construction is what I am unsure about:
why there is the `for item in items` line in this inner table since there is only one summary allowed per meeting minute item?

Reply:
It looks like you can have multiple `dd` without borking the list, so I've changed it to how you recommended.